src/main/java/core/basesyntax/service/impl/FruitTransactionMapper.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.service.Mapper;
+
+public class FruitTransactionMapper implements Mapper<FruitTransaction, String> {
+    private static final String DELIMITER = ",";
+    private static int OPERATION_INDEX = 0;
+    private static int FRUIT_INDEX = 1;
+    private static int QUANTITY_INDEX = 2;
```

Comment:
The fields `OPERATION_INDEX`, `FRUIT_INDEX`, and `QUANTITY_INDEX` should be final since they are constants (their values are not meant to be changed after initialization).